src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_RMSMappingQuality.java

```diff
@@ -51,7 +54,7 @@
     public List<VCFInfoHeaderLine> getDescriptions() {
         //TODO I'm just going to ignore this for now
```

Comment:
Can this TODO be addressed?

Reply:
Changing this to be more reasonable involves changing many integration test files. Now it is possible to request either a description or a raw description from the annotationEngine but I did not change it for this class. @droazen Should I try to update all the test files with the correct headers now?